### PR TITLE
Add Apollo to get open positions from graphcms

### PR DESCRIPTION
**Pull request by adrifdez**
Change the way that we are importing open positions from graphcms, instead of doing in the build and not be available to add more positions without a new build. We are loading the positions on the client side with 
Apollo client connected to graphql. This approach allow us reflect the changes from our cms in real time

### CODE REVIEW

File: apollo-client.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+
+export const client = new ApolloClient({
+  uri: process.env.GATSBY_GRAPHCMS_API_URL,
```
Comment (AlbertoElias) [inline apollo-client.js:4]: Why use the prefix gatsby instead of leaving the existing name?

File: src/pages/join-us/[position].jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      location
+      description
+      responsibilities
+      skills
+      niceToHave
+    }
+  }
+`
+export default function PositionPage ({ params }) {
+  console.log(params.position)
```
Comment (AlbertoElias) [inline src/pages/join-us/[position].jsx]: leftover log

File: src/pages/join-us/[position].jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  const { loading, error, data } = useQuery(POSITION, {
+    variables: { slug: params.position }
+  })
+  console.log(loading, error, data)
```
Comment (AlbertoElias) [inline src/pages/join-us/[position].jsx]: leftover

File: apollo-client.js (hunk 1 above)
Comment (adrifdez) [reply apollo-client.js:4]: To allow browser to access it
https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#accessing-environment-variables-in-the-browser

**Review by AlbertoElias — APPROVED** [review]

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**